test/src/integration/integration_test.go

```diff
@@ -159,20 +164,42 @@ func runAnsiblePlaybooks(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	err = ioutil.WriteFile("./inventory.ini", []byte(lookupTargetValue(t, "kubernetes", "inventory_ini")), 0644)
+	err = ioutil.WriteFile("./inventories/kubernetes", []byte(lookupTargetValue(t, "kubernetes", "inventory_ini")), 0644)
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	// Install tools
-	runAnsiblePlaybook(t, k8sModuleDir, []string{"./playbooks/playbook-install-tools.yml", "-e", "base_local_directory=../../../../", "-e", "install_awscli=" + installAwscli})
+	runAnsiblePlaybook(t, k8sModuleDir, "../inventories", []string{"./playbooks/playbook-install-tools.yml", "-e", "base_local_directory=../../../../", "-e", "install_awscli=" + installAwscli})
 
 	// Deploy scalardl
-	runAnsiblePlaybook(t, k8sModuleDir, []string{"./playbooks/playbook-deploy-scalardl.yml", "-e", "base_local_directory=../../../conf"})
+	runAnsiblePlaybook(t, k8sModuleDir, "../inventories", []string{"./playbooks/playbook-deploy-scalardl.yml", "-e", "base_local_directory=../../../conf"})
+}
+
+func runGoss(t *testing.T, targetModules []string, targetHosts string) {
+	cloudProvider := "aws"
+	if strings.Contains(*terraformDir, "azure") {
+		cloudProvider = "azure"
+	}
+
+	err := ioutil.WriteFile("./ssh.cfg", []byte(lookupTargetValue(t, "network", "ssh_config")), 0644)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for _, m := range targetModules {
+		err = ioutil.WriteFile("./inventories/"+m, []byte(lookupTargetValue(t, m, "inventory_ini")), 0644)
+		if err != nil {
+			t.Fatal(err)
+		}
+	}
+
+	// Ansible goss role
+	runAnsiblePlaybook(t, "./", "./inventories", []string{"../../modules/" + cloudProvider + "/network/.terraform/modules/network/provision/ansible/playbooks/goss-server.yml", "-l", targetHosts})
```

Comment:
To me, it feels somehow weird to go into the `.terraform` directory when running ansible. Wouldn't it be nice to have the goss playbook in this repo? Probably it is related to [my comment on scalar-terraform](https://github.com/scalar-labs/scalar-terraform/pull/282#pullrequestreview-593830056).

Reply:
I think the goss playbook defines the expected spec of scalar-terraform so they should be co-located.
(Otherwise, if we update an instance of the ledger in `scalar-terraform`, we need to update `scalar-terratest` repo as well.)

I agree it's a bit weird to use hidden files...

Reply:
`.terraform` is a working directory containing Terraform configuration files and ansible roles. So, I think it's ok to use it.

Reply:
I think it's not ideal, but there seems no better ways in this case.

Reply:
I think it's best to use the files under .terraform unless separate terraform and ansible.

The ansible roles that are copied to the bastion server also use the files under .terraform